### DISCUSSION TITLE
How to use ESModule in Deta

### DISCUSSION THREAD

**Discussion by jiesou** [deta/cloud-docs#345] [Help] [answered]
The error occurs when I try to use the `import` of ESModule, this is the log:
![image](https://user-images.githubusercontent.com/84175239/181312976-f7f6720d-3160-4846-9d87-b03861de2d91.png)

I have added `"type": "module"` in package.json and that doesn't work

**Comment by abdelhai** [comment, marked as answer]
hi, esmodules are not supported on deta yet. this is something we're working on at the moment
